main.tf

```diff
@@ -145,6 +145,15 @@ resource "aws_msk_cluster" "default" {
     storage_info {
       ebs_storage_info {
         volume_size = var.broker_volume_size
+
+        dynamic "provisioned_throughput" {
```

Comment:
In case we want to explicitly disable provisioned throughput, to produce for example:
```hcl
provisioned_throughput {
    enabled = false
}
```
the module with the change you suggested would try to set volume_throughput to the default value (250).

this would cover that case, and fix [this bug](https://github.com/cloudposse/terraform-aws-msk-apache-kafka-cluster/issues/99):

```hcl
        dynamic "provisioned_throughput" {
          for_each = var.broker_ebs_provisioned_throughput_enabled && var.broker_ebs_provisioned_throughput_state ? [1] : []

          content {
            enabled           = true
            volume_throughput = var.broker_ebs_provisioned_throughput_value
          }
        }

        dynamic "provisioned_throughput" {
          for_each = var.broker_ebs_provisioned_throughput_enabled && !var.broker_ebs_provisioned_throughput_state ? [1] : []

          content {
            enabled           = false
          }
        }

```

Reply:
@angryhamsterx  would you mind if we pull your PR changes into a seperate PR from us, with cloudposse'isms covered, and drive the PR forwards?

Reply:
@angryhamsterx  https://github.com/InfareSolutions/terraform-aws-msk-apache-kafka-cluster/tree/broker_ebs_provisioned_throughput